conda-recipe/meta.yaml

```diff
@@ -13,16 +13,11 @@ requirements:
   build:
     - python >=3.9
     - setuptools
+    - pip
   run:
     - python >=3.9
-    - pyqt5
-    - opencv
-    - numpy
-    - pathlib
-    - typing
-    - qasync
-    - qrcode
-    - qtawesome
+    - pip:
+        - -r requirements.txt
```

Comment:
Despite the repetition, it's generally better to keep the requirements explicitly listed here.  Conda has its own dependency resolution system, and using pip circumvents that.   The rule of thumb is to never mix conda and pip installs if you can help it.  

This is a packaging nitpick that you probably shouldn't have to care about, but figured it's worth a mention.